dist/index.js

```diff
@@ -23335,7 +23335,7 @@ class AppService {
                 installationId: installation.id,
                 appId,
                 roleRestrictions,
-                settings: Object.assign({ name: appConfig.name }, this.cleanParameters(params)),
+                settings: Object.assign({}, this.cleanParameters(params)),
```

Comment:
Aqui podemos remover o objeto vazio
```suggestion
                settings: Object.assign(this.cleanParameters(params)),
```